sys-block/parted/parted-3.2.recipe

```diff
@@ -77,36 +80,42 @@ BUILD_PREREQUIRES="
 	cmd:autoreconf
 	cmd:awk
 	cmd:gcc$secondaryArchSuffix
-	cmd:gettext
+	cmd:gettext$secondaryArchSuffix
 	cmd:gperf
 	cmd:grep
 	cmd:iconv
 	cmd:ld$secondaryArchSuffix
 	cmd:libtoolize$secondaryArchSuffix
 	cmd:make
-	cmd:makeinfo
 	cmd:pkg_config$secondaryArchSuffix
 	cmd:sed
 	"
 
+TEST_REQUIRES="
+	cmd:dd
+	"
+
 defineDebugInfoPackage parted$secondaryArchSuffix \
 	"$libDir"/libparted.so.$libVersion \
 	"$libDir"/libparted-fs-resize.so.$libFsResizeVersion
 
 BUILD()
 {
 	autoreconf
+	MAKEINFO=true \
 	runConfigure --omit-dirs sbinDir ./configure --sbindir=$commandBinDir \
 		--disable-device-mapper
-	make $jobArgs
+	make $jobArgs \
+		CPPFLAGS="-Wimplicit-fallthrough=0" \
+		LDFLAGS="-lblkid -luuid -lintl" \
```

Comment:
The backslash isn't needed.

Reply:
The reason why I added a backslash at the end of this line is because:
* I needed to change that line anyway;
* I want to make this command easier to modify: if we ever need to pass more parameters to `make` then the `git log -p` will look nicer since we won't have a `-` in front of that line, with a `+` in front of an almost identical line, just below.

So, although I know the backslash is not needed, this is a style issue, and I'd like to keep it unchanged.

Thanks for the review, anyway :)

Reply:
This doesn't make much sense, but thanks for the answer!

Reply:
Well, it looks like I'm not the only one to think it _does_ make sense :smile:
<Details>

https://github.com/haikuports/haikuports/blob/fc0936b04009e713f9bf551851781c83f9f23871/app-pda/libplist/libplist-1.12.recipe#L93
https://github.com/haikuports/haikuports/blob/070bb79d761a32dcfd1814895cc441813458766e/dev-lang/vala/vala-0.40.6.recipe#L96
https://github.com/haikuports/haikuports/blob/4ceccd41e14e8a55d3d6d0a9ed80fd5c95840934/dev-libs/libzip/libzip-0.10.1.recipe#L68
https://github.com/haikuports/haikuports/blob/4ceccd41e14e8a55d3d6d0a9ed80fd5c95840934/dev-libs/libzip/libzip-0.11.2.recipe#L68
https://github.com/haikuports/haikuports/blob/dd081cc943be58879c3dff19d81b4962f9224d92/dev-python/sip/sip-4.19.5.recipe#L72
https://github.com/haikuports/haikuports/blob/dd081cc943be58879c3dff19d81b4962f9224d92/dev-python/sip/sip-4.19.5.recipe#L89
https://github.com/haikuports/haikuports/blob/b242e15ec42bd407aa25ab9730521b44592c555f/kde-frameworks/kdelibs4support/kdelibs4support-5.46.0.recipe#L168
https://github.com/haikuports/haikuports/blob/3983b6329a22b392c6bde2328c14aed31c729776/media-libs/libebur128/libebur128-1.2.3.recipe#L57
https://github.com/haikuports/haikuports/blob/936a20609127e2ff8d400ee107021227f8aeb889/media-libs/libvorbis/libvorbis-1.3.6.recipe#L76
https://github.com/haikuports/haikuports/blob/69769a3ac56e8829e3445cca7edb8b27a66cdda7/media-libs/libvpx/libvpx-1.7.0.recipe#L97
https://github.com/haikuports/haikuports/blob/fc0936b04009e713f9bf551851781c83f9f23871/sci-libs/superlu/superlu-5.2.1.recipe#L65
https://github.com/haikuports/haikuports/blob/f95f57441d394071c30cedcd2b6c19ee9fbb3bbe/sci-visualization/gnuplot/gnuplot-5.0.7.recipe#L93
https://github.com/haikuports/haikuports/blob/9001de1a37ebac26085cb6fab7c9c71c0154f7f5/sci-visualization/gnuplot/gnuplot-5.2.3.recipe#L77
https://github.com/haikuports/haikuports/blob/4634abef1be46823fc913ed96d09fb25ed5f519b/sys-apps/coreutils/coreutils-8.22.recipe#L251
https://github.com/haikuports/haikuports/blob/4634abef1be46823fc913ed96d09fb25ed5f519b/sys-apps/coreutils/coreutils-8.23.recipe#L251
https://github.com/haikuports/haikuports/blob/40ae423b02003e813effd928bba4d51e8fd1f564/sys-apps/coreutils/coreutils-8.24.recipe#L250
https://github.com/haikuports/haikuports/blob/098fcc483b16e8ddc24985a0f53bdd652ab0d708/sys-apps/coreutils/coreutils-8.29.recipe#L246

</Details>